back/boxtribute_server/business_logic/box_transfer/agreement/crud.py

```diff
@@ -57,9 +57,34 @@ def create_transfer_agreement(
     Raise an InvalidTransferAgreementBase expection if any specified source/target base
     is not part of the source/target organisation.
     """
-    if source_organisation_id == target_organisation_id:
+    if initiating_organisation_id == partner_organisation_id:
         raise InvalidTransferAgreementOrganisation()
 
+    # In GraphQL input, partner organisation base IDs can be omitted, hence substitute
+    # actual base IDs of partner organisation. Avoid duplicate base IDs by creating sets
+    initiating_organisation_base_ids = set(initiating_organisation_base_ids)
+    if partner_organisation_base_ids is None:
+        partner_organisation_base_ids = [
+            b.id
+            for b in Base.select().where(Base.organisation == partner_organisation_id)
```

Comment:
@pylipp I believe we need to add a task in trello. There are inactive bases in the db that no user has access to anymore, e.g. Drops has the base Lesvos. We probably need to add a column in the base db table to mark inactive bases.

Reply:
let me know if you have logged this somewhere, even if it's just a link to this comment